experiments/experiments-impl/src/main/java/com/duckduckgo/experiments/impl/VariantManagerImpl.kt

```diff
@@ -44,7 +43,7 @@ class VariantManagerImpl @Inject constructor(
 
     @Synchronized
     override fun getVariantKey(): String {
-        val activeVariants = ACTIVE_VARIANTS
+        val activeVariants = convertEntitiesToVariants(experimentVariantRepository.getActiveVariants())
```

Comment:
Why are we not making `getVariantKey()` a `suspend` function and also directly set an `io()` context dispatcher here?

Reply:
That was my first implementation, however, if I make all the way up suspend, there is a call inside an Observable flatmap which I didn't know how to resolve and I decided to "trust" we'll be calling it from the background thread. 
Let's talk about this offline to make a decision.